svcb.go

```diff
@@ -522,42 +537,42 @@ func (s *SVCBIPv4Hint) copy() SVCBKeyValue {
 	}
 }
 
-// SVCBECHConfig pair contains the ECHConfig structure defined in draft-ietf-tls-esni [RFC xxxx].
-// Basic use pattern for creating an echconfig option:
+// SVCBECH pair contains the ECHConfig structure defined in draft-ietf-tls-esni [RFC xxxx].
+// Basic use pattern for creating an ech option:
 //
 //	h := new(dns.HTTPS)
 //	h.Hdr = dns.RR_Header{Name: ".", Rrtype: dns.TypeHTTPS, Class: dns.ClassINET}
-//	e := new(dns.SVCBECHConfig)
+//	e := new(dns.SVCBECH)
 //	e.ECH = []byte{0xfe, 0x08, ...}
 //	h.Value = append(h.Value, e)
-type SVCBECHConfig struct {
-	ECH []byte
+type SVCBECH struct {
+	ECHConfigList []byte // includes the redundant length prefix
```

Comment:
this is too long.  Just `Config`?  What does the draft use?

Reply:
The draft uses `ECHConfigList`